README.md

```diff
@@ -1,3 +1,53 @@
-# bulk-scan-orchestrator
+# Bulk Scan Orchestrator
 
 [![Build Status](https://travis-ci.org/hmcts/bulk-scan-orchestrator.svg?branch=master)](https://travis-ci.org/hmcts/bulk-scan-orchestrator)
+
+## Purpose
+
+The purpose of this application is to retrieve information about envelopes uploaded to Bulk Scanning
+(via Azure Service Bus queue) and manage the related CCD cases.
```

Comment:
Isn't it `uploaded to Azure Blob Storage` or `uploaded to HMCTS Document Store via Bulk Scan Processor`?

Reply:
Bulk scanning orchestrator does below 
1. Consume SB message and retrieve Envelope information using envelope id.
2. Based on envelope metadata create an exception record or updates an existing case in CCD.
3. Send envelope process event to bulk scanning processor.
4. Handle failed event(to be sent to bsp), failed envelope retrieval ,failed case/exception records.